tests/services/test_cluster.py

```diff
@@ -81,14 +81,14 @@ async def start(self):
 )
 @pytest.mark.asyncio
 @pytest.mark.skipif(sys.version_info < (3, 8), reason="AsyncMock requires python 3.8+")
-async def test_spawn_instances(kwargs, args, error, monkeypatch):
+async def test_spawn_instances(kwargs, args, error, monkeypatch, golem_factory):
```

Comment:
again, had we used a factory_boy factory, we could parametrize the factory to e.g. apply that `asyncmock` to `_get_new_engine` when needed ... anyway, this is completely optional... just a thought